python/pyspark/sql/connect/proto/types_pb2.pyi

```diff
@@ -48,13 +48,15 @@ else:
 
 DESCRIPTOR: google.protobuf.descriptor.FileDescriptor
 
+@typing_extensions.final
```

Comment:
These changes might be due to difference in python code generator.
Cc: @HyukjinKwon (are we planning to generated these at build time to avoid issues like this?).

Reply:
That's exactly what I have been asking around .. but seems that's a bit difficult.

Reply:
cc @grundprinzip and @zhengruifeng FYI

Reply:
I think I saw similar annotation generated in [another PR](https://github.com/apache/spark/pull/40297#discussion_r1133474707), but it was removed in some way then in that PR.